SUI/SUI/ContentView.swift

```diff
@@ -7,18 +7,129 @@
 
 import SwiftUI
 
+/// Экран с кошельками
 struct ContentView: View {
+    private enum Constants {
+        static let toggleText = "Показать настройки"
+        static let roundedRectangleCornerRadius = 10
+        static let someText = "Что-то на экране"
+        static let isXOffsetChange = 100
+        static let originalXOffset = 0
+        static let firstWalletText = "Кошелек 1"
+        static let secondWalletText = "Кошелек 2"
+        static let profileText = "Профиль"
+        static let spacerHeight = 400
+        static let response = 0.5
+        static let dampingFraction = 0.7
+        static let blendDuration = 0.3
+        static let cashInButtonName = "Пополнить баланс"
+        static let openAnAccountButtonName = "Открыть счет"
+        static let openAnAccountAlertTitle = "Открыть счет?"
+        static let transferButtonName = "Перевести со счета на счет"
+        static let replenishButtonName = "Пополнить"
+        static let balanceConfDialogText = "Пополнить счет?"
+        static let primaryButtonOpenAccountText = "Пополнить"
+        static let transferAlertTitle = "Перевести 300 у.е с кошелька 1 на кошелек 2?"
+        static let transferButtonTitle = "Перевести"
+        static let zeroOpacity = 0
+        static let oneOpacity = 1
+        static let balance = 300
+    }
+    
+    @State private var isOnToggle = false
+    @State private var isBalanceButtonIsTapped = false
+    @State private var isOpenAnAccountButtonTapped = false
```

Comment:
```suggestion
    @State private var isBalanceButtonShown = false
    @State private var isOpenAnAccountButtonShown= false
```
если флаг отвечает за показ чего-либо, то на конце Shown